app.py

```diff
@@ -0,0 +1,37 @@
+from flask import Flask, redirect, request
+from json import dumps
+from urllib.parse import urlencode
+
+app = Flask(__name__)
+app.debug = True
+
+steam_openid_url = 'https://steamcommunity.com/openid/login'

```

Comment:
The use of a constant helps to avoid URL repetitions, so it can be said that the author follows the principles of Dry programming.